constraintlayout/core/src/main/java/androidx/constraintlayout/core/motion/MotionWidget.java

```diff
@@ -70,8 +70,8 @@ public static class Motion {
         public String mQuantizeInterpolatorString = null;
         public int mQuantizeInterpolatorType = INTERPOLATOR_UNDEFINED; // undefined
         public int mQuantizeInterpolatorID = -1;
-        @SuppressWarnings("unused") private static final int INTERPOLATOR_REFERENCE_ID = -2;
-        @SuppressWarnings("unused") private static final int SPLINE_STRING = -1;
```

Comment:
Likewise, they remain unused.

Reply:
This was a rebase accident removing the changes